spec/gemstash/upstream_spec.rb

```diff
@@ -55,19 +65,43 @@
     expect(upstream_uri.url("gems", "key=value")).to eq("https://rubygems.org/gems?key=value")
   end
 
-  it "fails if the uri is not valid" do
-    expect { Gemstash::Upstream.new("something_that_is_not_an_uri") }.to raise_error(
-      /URL 'something_that_is_not_an_uri' is not valid/
-    )
-  end
-
   it "has a nil user agent if not provided" do
     expect(Gemstash::Upstream.new("https://rubygems.org/").user_agent).to be_nil
   end
 
   it "supports getting user agent" do
     expect(Gemstash::Upstream.new("https://rubygems.org/",
-                                  user_agent: "my_user_agent").user_agent).to eq("my_user_agent")
+      user_agent: "my_user_agent").user_agent).to eq("my_user_agent")
```

Comment:
RuboCop complains about this line, perhaps you can autofix it, perhaps you can return it to what it was before?

Reply:
Sorry, an automatic "fix" from my editor... 🤦‍♂️ It is fixed now.